GenderPayGap.WebUI/Models/ReportStarting/ReportStartingViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace GenderPayGap.WebUI.Models.ReportStarting
+{
+    public class ReportStartingViewModel
+    {
+        [BindNever /* Output Only - only used for sending data from the Controller to the View */]

```

Comment:
The [BindNever] attributes aren't needed here since this view model isn't used for a form submission